AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -3,104 +3,367 @@
+        from .tests.rigid_body import RigidBody_MaxAngularVelocityWorks as test_module
+
+    class Joints_HingeSoftLimitsConstrained(EditorSharedTest):
+        from .tests.joints import Joints_HingeSoftLimitsConstrained as test_module
+
+    class Joints_BallSoftLimitsConstrained(EditorSharedTest):
+        from .tests.joints import Joints_BallSoftLimitsConstrained as test_module
+
+    class Joints_BallLeadFollowerCollide(EditorSharedTest):
+        from .tests.joints import Joints_BallLeadFollowerCollide as test_module
+        
+    class ForceRegion_WorldSpaceForceOnRigidBodies(EditorSharedTest):
+        from .tests.force_region import ForceRegion_WorldSpaceForceOnRigidBodies as test_module
+        
+    class ForceRegion_PointForceOnRigidBodies(EditorSharedTest):
+        from .tests.force_region import ForceRegion_PointForceOnRigidBodies as test_module
+        
+    class ForceRegion_SphereShapedForce(EditorSharedTest):
+        from .tests.force_region import ForceRegion_SphereShapedForce as test_module
+        
+    class ForceRegion_RotationalOffset(EditorSharedTest):
+        from .tests.force_region import ForceRegion_RotationalOffset as test_module
+
+    class RigidBody_EnablingGravityWorksPoC(EditorSharedTest):
+        from .tests.rigid_body import RigidBody_EnablingGravityWorksPoC as test_module
+
+    class Collider_ColliderRotationOffset(EditorSharedTest):
+        from .tests.collider import Collider_ColliderRotationOffset as test_module
+
+    class ForceRegion_ParentChildForcesCombineForces(EditorSharedTest):
+        from .tests.force_region import ForceRegion_ParentChildForcesCombineForces as test_module
+
+
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomationNoPrefab(EditorTestSuite):
+
+    global_extra_cmdline_args = ["-BatchMode", "-autotest_mode",
+                                 '--regset=/Amazon/Preferences/EnablePrefabSystem=false']
+
+    @staticmethod
+    def get_number_parallel_editors():
+        return 16
+
+    @pytest.mark.xfail(reason="AssertionError: No run data for test: Physics_UndoRedoWorksOnEntityWithPhysComponents.")
+    class Physics_UndoRedoWorksOnEntityWithPhysComponents(EditorSharedTest):
+        from .tests import Physics_UndoRedoWorksOnEntityWithPhysComponents as test_module
+
+    @pytest.mark.xfail(reason="AssertionError: Failed to open level: ScriptCanvas_ShapeCast does not exist or is invalid")
+    class ScriptCanvas_ShapeCast(EditorSharedTest):
+        from .tests.script_canvas import ScriptCanvas_ShapeCast as test_module
+
+    @pytest.mark.xfail(reason="AssertionError: Failed to open level: ForceRegion_SliceFileInstantiates does not exist or is invalid")
+    class ForceRegion_SliceFileInstantiates(EditorSharedTest):
+        from .tests.force_region import ForceRegion_SliceFileInstantiates as test_module
```

Comment:
All these tests are failing and I've added the specific reason on each one. 

Before it was the whole `class TestAutomationNoPrefab` the one that had xfail, so all the tests could fail and not complain. I've changed it so each tests now has the specific reason why it fails.

This will have to be addressed at later.